docs/admin/automation.md

```diff
@@ -23,14 +19,13 @@ coder workspaces ls
 
 ## REST API
 
-You can use tokens with the Coder's REST API using the `Coder-Session-Token` HTTP header.
+You can review the [API reference](../api/index.md) to find the necessary routes and payload. Alternatively, you can enable the [Swagger](https://swagger.io/) endpoint to play with the API:
```

Comment:
I'd like to avoid the word "play" here since we're doing actual requests against the API which modify the state.

```suggestion
You can review the [API reference](../api/index.md) to find the necessary routes and payload. Alternatively, you can enable the [Swagger](https://swagger.io/) endpoint to read the documentation and do requests against the API:
```